cmd/backfill/main.go

```diff
@@ -169,16 +169,16 @@ const (
 	colBizOwn     = 14 // O
 	colPrjNum     = 15 // P
```

Comment:
This const is unused in this file, which golangci-lint picks up as an error, which fails the lint job.

Reply:
Removed it